includes/templates/story-templates/fandom-title.html

```diff
@@ -1,6 +1,6 @@
 <!-- wp:amp/amp-story-page {"mediaType":"image","backgroundColors":"[{\u0022color\u0022:\u0022#313131\u0022}]","overlayOpacity":55} -->
 <amp-story-page style="background-color:#ffffff" id="038df815-5dc9-4105-bbca-77242ca79085" class="wp-block-amp-amp-story-page"><amp-story-grid-layer template="fill"><amp-img layout="fill" src="AMP_STORY_TEMPLATES_URL/fandom-title.jpg" style="object-position:initial"></amp-img></amp-story-grid-layer><amp-story-grid-layer template="fill" style="background-color:#313131;opacity:0.55"></amp-story-grid-layer>
-    <!-- wp:amp/amp-story-text {"type":"p","className":"is-style-rounded","customTextColor":"#ffffff","customBackgroundColor":"#e44240","autoFontSize":11,"positionTop":52.26,"positionLeft":6.1,"height":30,"width":111} -->
+    <!-- wp:amp/amp-story-text {"type":"p","className":"is-style-rounded","customTextColor":"#ffffff","customBackgroundColor":"#e44240","autoFontSize":10,"positionTop":52.26,"positionLeft":6.1,"height":30,"width":111} -->
     <amp-story-grid-layer data-block-name="amp/amp-story-text" template="vertical"><div class="amp-story-block-wrapper" style="position:absolute;top:52.26%;left:6.1%;width:33.84%;height:5.42%"><p style="background-color:rgba(228, 66, 64, 1);color:#ffffff;display:flex" class="wp-block-amp-amp-story-text is-style-rounded has-text-color has-background is-style-rounded" id="1bc21884-841c-4218-9e4e-a1568d3e47ae"><amp-fit-text layout="flex-item" class="amp-text-content">SPOILERS ALERT</amp-fit-text></p></div></amp-story-grid-layer>
```

Comment:
This is a very minor change to prevent the 'SPOILER ALERTS' text from being on 2 lines instead of 1.